cityelf/src/main/java/com/cityelf/service/UserService.java

```diff
@@ -0,0 +1,48 @@
+package com.cityelf.service;
+
+import com.cityelf.model.User;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserService {
```

Comment:
Add another class UserRepository which populates users and put it in **repository** folder
This class only calls a method getUsers from repo class